package.json

```diff
@@ -88,12 +89,11 @@
     "jsdom": "^11.6.2",
     "lint-staged": "^4.1.1",
     "match-media-mock": "^0.1.1",
-    "prettier": "^1.15.3",
+    "prettier": "^2.0.5",
```

Comment:
Doesn't balena/lint include prettier as well?

Reply:
It does which is why I updated this to match, the problem is this repo uses eslint for js files with quite a specialized config, so whilst I could switch to balena-lint for the js I don't know enough about the special stuff this repo uses to judge if it's safe and so we need to keep prettier to do `prettier && eslint`

Reply:
@Page- we should define react-specific linting rules in @balena/lint and remove everything custom here. I have an issue in the dashboard for it already, but haven't found the time to do it, so for now let's just merge this as it is.